maestro-cli/src/main/java/maestro/cli/util/TimeUtils.kt

```diff
@@ -0,0 +1,17 @@
+package maestro.cli.util
+
+import kotlin.math.roundToLong
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+
+object TimeUtils {
+
+    fun durationInSeconds(startTime: Long?, endTime: Long?): Duration {
```

Comment:
let's also be specific that the start and end time should be in milliseconds.
```
durationInSeconds(startTimeInMillis: Long?, endTimeInMillis: Long?)
```

Reply:
applied